dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java

```diff
@@ -371,28 +372,28 @@ void testTestJson() {
 
         // Update tag with test parameters
         Client.main(new String[] { CONFIG, ResourceHelpers.resourceFilePath("config_file.txt"), TOOL, TEST_PARAMETER, ENTRY,
-            "quay.io/dockstoretestuser/test_input_json", VERSION, "master", "--descriptor-type", "cwl", "--add", "test.cwl.json",
+            "quay.io/dockstoretestuser/test_input_json", VERSION, "master", "--descriptor-type", CWL.toString(), "--add", "test.cwl.json",
```

Comment:
Some of these `toString()` may be redundant. 
Couldn't find a better link, but https://youtrack.jetbrains.com/issue/IDEA-265548 gives the general idea.

Reply:
I removed the redundant `toString()`s. 
[97c4b23](https://github.com/dockstore/dockstore-cli/pull/232/commits/97c4b23d0ac7924860fb328999bea8e84135fbbe) 
[6c059bd](https://github.com/dockstore/dockstore-cli/pull/232/commits/6c059bd3cb7cce326248eb7b49e4f953b74c86f4)

In the case of the selected code, it appears that the `.toString()` is required, as if I remove it, I get this error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:testCompile (default-testCompile) on project dockstore-cli-integration-testing: Compilation failure
[ERROR] /home/fhembroff/Documents/work/extract-cwl-wdl-to-constants/dockstore-cli/dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java:[374,98] incompatible types: io.dockstore.common.DescriptorLanguage cannot be converted to java.lang.String
[ERROR] 

```